components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/internal/TenantManagementListener.java

```diff
@@ -66,7 +66,7 @@ public void onTenantInitialActivation(int tenantId) throws StratosException {
         String tenantDomain = IdentityTenantUtil.getTenantDomain(tenantId);
         carbonContext.setTenantId(tenantId);
         carbonContext.setTenantDomain(tenantDomain);
-        CertificateValidationUtil.addDefaultValidationConfigInRegistry(tenantDomain);
+        CertificateValidationUtil.addDefaultValidationConfigInRegistry();
```

Comment:
Better to pass the `tenantDomain` since this method is adding the default configs for a specific tenant..

Reply:
Its true that, config store based impl doesn't need the `tenantDomain`.. But IMO, its a implementation detail.